src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -352,6 +353,15 @@ public String resolveForStrings(String content) {
     }
 
     /**
+     * Resolves any references to strings contained in this field content,
+     * if possible.
+     */
+    public String resolveForStrings(Optional<String> content) {
```

Comment:
Me neither, but at the moment it looked like the better option (compared to blindly doing `get()` or explicitly adding `ifPresent()`). I realized another one now though.